full-service/src/json_rpc/transaction_log.rs

```diff
@@ -31,7 +27,7 @@ pub struct TransactionLog {
     pub input_txos: Vec<InputTxo>,
 
     /// A list of the Txos which were outputs from this transaction.
-    pub output_txos: Vec<OutputTxo>,
+    pub payload_txos: Vec<OutputTxo>,
```

Comment:
This is potentially a breaking API change. Can we move these changes to another PR and specify the migration plan?